### PR TITLE
Added eWay Direct

### PR DESCRIPTION
**Pull request by alexw23**
Everything is working great, tests are completed and I have now squashed into a single commit for readability.

Ready for merge @adrianmacneil :+1: :package:

### CODE REVIEW

**Comment by Mihai-P** [comment]
Hi guys, I want to add some more functionality to the Eway Direct option. Like the possibility to capture a card, update a card, make a payment with a captured card etc. Should I wait for this to be merged before I start developing?

File: src/Message/DirectCaptureRequest.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        $sxml = new \SimpleXMLElement($xml);
+
+        /* eWAY Customer Id */
+        $sxml->addChild('ewayCustomerID', $this->getCustomerId());
+
+        /* eWAY Transaction Details */
+        $sxml->addChild('ewayTotalAmount', $this->getAmountInteger());
+        $sxml->addChild('ewayAuthTrxnNumber', $this->getTransactionId());
+
+        $sxml->addChild('ewayCardExpiryMonth', null); // optional field
```
Comment (kayladnls) [inline src/Message/DirectCaptureRequest.php]: If these are optional, should they be sent at all?

**Comment by kayladnls** [comment]
@alexw23 This looks good, save for that single question I had. Should be able to merge this in soon. 

@Mihai-P You can make your PR now if you're ready. If not, this branch should be merged in soon.

**Comment by Mihai-P** [comment]
@kayladnls I want to build on top of what alexw23 did, I already have it up as a package but only the Credit cart management stuff, so together with that part would make a great package. I will wait for the merge.

**Comment by kayladnls** [comment]
@alexw23 any movement in this? Also, can you remove DS_STORE?

File: src/Message/DirectCaptureRequest.php (hunk 1 above)
Comment (alexw23) [inline src/Message/DirectCaptureRequest.php]: eWay is a little fussy about XML validation. It's better to have it in there as they may change spec later on down the track.

**Comment by alexw23** [comment]
@kayladnls done. See feedback on optional field. I think it's time to merge it in :+1:

**Comment by alexw23** [comment]
@Mihai-P when you do can you tag me in so I can see what's going on :+1:

**Comment by ecdeploy** [comment]
Is this going to be merged?

**Comment by alexw23** [comment]
Thanks @kayladnls can you also merge PR #6

**Pull request merged**